### DISCUSSION TITLE
It possible to run two shcedulers in paralell

### DISCUSSION THREAD

**Discussion by danies8** [quartznet/quartznet#2259] [Q&A]
Hi,
Is it  possible to run two shcedulers in paralell in the same time ?
Thanks,

**Comment by viveknuna** [comment]
@danies8 do you mean multiple schedulers on the same database or different-2 databases?

@jafin do you have any inputs on this?

**Comment by danies8** [comment]
I used my db mongo not sql not connected to scheduler..
This are my configurations to work with my schedulers.

 _logger.Debug("Create scheduler");
 _scheduler = await _stdSchedulerFactory.GetScheduler();

 _logger.Debug("Create sacn info scheduler");
 _scanHistoryscheduler = await _scanHistoryStdSchedulerFactory.GetScheduler();



 _logger.Debug("Config default routes data.");
//For case of first time we add / route and we want to configure it
await ScheduleJobs();
await _scheduler.Start();//token

_logger.Debug("Config the scan history scheduler with routes data once.");
await ScheduleScanHistoryJob();
_logger.Debug("Run the scan history scheduler");
await _scanHistoryscheduler.Start();

**Comment by danies8** [comment]
@viveknuna do you have an answer?
